task-definition.plugins-async.json

```diff
@@ -115,6 +115,10 @@
                 {
                     "name": "WORKER_CONCURRENCY",
                     "value": "2"
+                },
+                {
+                    "name": "INGESTION_BATCH_BREAKUP_BY_DISTINCT_ID_TEAMS",
```

Comment:
We don't need this in this file.

Reply:
assuming we don't do future trickery and run ingestion there 😅 - I wonder if we should have a plugin-server common config and then async and ingestion separate parts, which would make it easier to see stuff from quick looks

Reply:
Given that this will only be in for a short time anyway, I removed this.